lockable-object/README.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: pattern
+title: Lockable Object
+folder: lockable-object
+permalink: /patterns/lockable-object/
+categories: Concurrency
+tags:
+- Performance
+---
+
+
+## Intent
+The lockable object design pattern ensures that there is only one user using the target object. Compared to the built-in synchronization mechanisms such as using the `synchronized` keyword, this pattern can lock objects for an undetermined time and is not tied to the duration of the request.
+
+## Explanation
```

Comment:
@ohbus Thanks!
There is an Explanation subject in this README.md file.
Is there anything wrong with it? it looks like it has the same structure as the examples you have shared.

Reply:
I can understand that and would love if you could **expand** on it as it has already been done in those patterns like
- [Explanation](https://github.com/iluwatar/java-design-patterns/tree/master/flyweight#explanation)
- [Applicability](https://github.com/iluwatar/java-design-patterns/tree/master/flyweight#applicability)
- [Real world examples](https://github.com/iluwatar/java-design-patterns/tree/master/flyweight#real-world-examples)
- [Credits](https://github.com/iluwatar/java-design-patterns/tree/master/flyweight#credits)

It is not mandatory, but we would love to see it there if you can.
Please do let us know if you are willing to work on them or not, we'll decide accordingly.

Thanks 😄

Reply:
@ohbus Oh ok I got you, yea sure I will dive into it later on :)

Reply:
@ohbus Done.